src/main/scala/io/lenses/connect/secrets/providers/VaultSecretProvider.scala

```diff
@@ -162,4 +168,20 @@ class VaultSecretProvider() extends ConfigProvider with VaultHelper {
         )
     }
   }
+
+  private def readSecretFromVault(path: String): LogicalResponse = {
+    // Check if the path looks like it's referencing the vault database engine. If so, use the `database()` api of the
```

Comment:
This comment can be removed I think, it states what the code does line by line without adding value.